script.js

```diff
@@ -403,10 +486,12 @@ require(['jquery','core/str'], function ($, Str)
                 this.hide();
 
                 function move(e) {
+
                     var m = $(e.target).closest('a').attr('class').match(/move-(\d+)-to-(\d+)/);
```

Comment:
Be more specific - what type of id's?

If I later on want to use a course id, could I use id or is it a section id or a cmid. Maybe call them section_id_from and section_id_to.